lib/ansible/cli/__init__.py

```diff
@@ -597,8 +597,7 @@ def parse(self):
 
             else:
                 # set default if it exists
-                if os.path.exists(C.DEFAULT_HOST_LIST):
-                    self.options.inventory = [C.DEFAULT_HOST_LIST]
+                self.options.inventory = [host_list for host_list in C.DEFAULT_HOST_LIST if os.path.exists(host_list)]
```

Comment:
this would ignore missing inventory files

Reply:
wanted to keep the logic as is. Error out if path not os.path.exists?

Reply:
ok, there is already some code that puts a warning to the user. " [WARNING]: Unable to parse /path/hosts/foo as an inventory source"